tests/mqtt-limit-log-3/test.yaml

```diff
@@ -0,0 +1,19 @@
+pcap: ../mqtt5-sub-userpass/input.pcap
+
+requires:
+  min-version: 8
+
+args:
+  - -k none
+
+checks:
+
+  - filter:
+      count: 1
+      match:
+        event_type: mqtt
+        mqtt.subscribe.qos: 1
+        mqtt.subscribe.retain: false
+        mqtt.subscribe.dup: false
+        mqtt.subscribe.topics: [ {topic: topicX, qos: 0}, {topic: "topi[truncated 2 additional bytes]", qos: 0} ]
```

Comment:
Nit : Should we test `mqtt.subscribe.topics[1]` ? and `mqtt.subscribe.topics.__len: 2`

Reply:
Doesn't line 18 test for exact equality of the result? The code in `FilterCheck::match()` seems to indicate so. Then I don't think there is a reason to add these two tests, because both the existence of `mqtt.subscribe.topics[1]` as well as the array length is already covered in the line above.

Reply:
Yes it works.
This is a nit style remark...